mmcv/image/io.py

```diff
@@ -61,7 +62,7 @@ def imread(img_or_path, flag='color', channel_order='bgr'):
     """Read an image.
 
     Args:
-        img_or_path (ndarray or str): Either a numpy array or image path.
+        img_or_path (ndarray or str or :obj:`Path`): Either a numpy array or str or pathlib.Path.
             If it is a numpy array (loaded image), then it will be returned
             as is.
```

Comment:
This line is so long that it caused flake8 error in CI. 
You may change it to:
```python
img_or_path (ndarray or str or :obj:`Path`): Either a numpy array or str 
    or pathlib.Path.If it is a numpy array (loaded image), then it will 
    be returned as is.
```